command/testdata/apply/output.jsonlog

```diff
@@ -1,5 +1,6 @@
 {"@level":"info","@message":"Terraform 0.15.0-dev","@module":"terraform.ui","terraform":"0.15.0-dev","type":"version","ui":"0.1.0"}
 {"@level":"info","@message":"test_instance.foo: Plan to create","@module":"terraform.ui","change":{"resource":{"addr":"test_instance.foo","module":"","resource":"test_instance.foo","implied_provider":"test","resource_type":"test_instance","resource_name":"foo","resource_key":null},"action":"create"},"type":"planned_change"}
+{"@level":"info","@message":"Plan: 1 to add, 0 to change, 0 to destroy.","@module":"terraform.ui","changes":{"add":1,"change":0,"remove":0,"operation":"plan"},"type":"change_summary"}
```

Comment:
This new message surprised me at first, but I guess it's just a consequence of the plan rendering being restructured. My first concern was that it would make integration more difficult, but fortunately the `changes` object includes an `operation` field that I had forgotten about, so I think this is just fine.

I believe that it also won't affect integrations which use a saved plan, which was my main concern anyway.

Reply:
Yes, this new message is the JSON equivalent of a new behavior in the human `apply` output where Terraform repeats the plan that it's about to apply, because subjectively I found it jarring to have Terraform just immediately start doing something without explaining what it was doing, _particularly_ in the case where there are no actions at all and thus updating the state was the only side-effect, where the output in that case made it seem like Terraform had done nothing at all.

While introducing this into the JSON output wasn't my original intent, once I figured out what I'd done and why it was happening it retroactively seemed justified, because a hypothetical custom UI around this output could for example choose to summarize that only 5 of 6 planned additions happened, for example. (Admittedly that would also be possible by parsing the detailed plan, but that's also true for the _apply_ summary and so it seemed like making this information conveniently available was the intent.)